commit bb78ff2cb313bd7ad1d13b6f1519fb547aacbe7a
Author: David Faure <faure@kde.org>
Date:   Wed Jun 26 19:02:37 2013 +0200

    Restore the space between the selected filenames.
    
    Was a bug in my previous commit to this file.
    FIXED-IN: 4.10.5
    BUG: 321642

diff --git a/kfile/kfilewidget.cpp b/kfile/kfilewidget.cpp
index fffe753..0d55fb3 100644
--- a/kfile/kfilewidget.cpp
+++ b/kfile/kfilewidget.cpp
@@ -1254,7 +1254,7 @@ void KFileWidgetPrivate::setLocationText( const KUrl::List& urlList )
     if ( urlList.count() > 1 ) {
         QString urls;
         foreach (const KUrl &url, urlList) {
-            urls += QString("\"%1\"").arg(relativePathOrUrl(currUrl, url));
+            urls += QString("\"%1\"").arg(relativePathOrUrl(currUrl, url)) + ' ';
         }
         urls = urls.left( urls.size() - 1 );
 
